taxonomy-editor-frontend/src/pages/home/index.jsx

```diff
@@ -0,0 +1,83 @@
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import Link from '@mui/material/Link';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import { useNavigate } from "react-router-dom";
+
+function Copyright(props) {

```

Comment:
if this component is only used here, what about not passing props and defining everything on it? passing props and descructuring this way becomes a bit difficult to follow at some point